datasets/Makefile

```diff
@@ -0,0 +1,167 @@
+# Copyright (c) 2020-2021 The MLPerf Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+##########################################################################
+
+# Get absolute path of makefile
+MK_PATH := $(abspath $(lastword $(MAKEFILE_LIST)))
+
+# Specify directories
+ROOTDIR := $(abspath $(shell dirname ${MK_PATH})/..)
+BUILDDIR := ${ROOTDIR}/datasets
+
+.PHONY: clean clean-ade20k clean-coco clean-imagenet clean-squad
+
+USERID = $(shell id -u)
+GROUPID = $(shell id -g)
+
+# Ade20k calibration vars
+A_SRC_DIR := ${BUILDDIR}/downloads/ADEChallengeData2016/images/training
+A_FILES = $(shell cat ${BUILDDIR}/util/ade20k/ade20k_calibration_files.txt)
+A_SRC = $(addprefix $(A_SRC_DIR)/,$(A_FILES))
+A_DEST_DIR := ${BUILDDIR}/output/calibration_set/ade20k/images
+
+A_TARGS := $(patsubst $(A_SRC_DIR)/%.jpg,$(A_DEST_DIR)/%.jpg,$(A_SRC))
```

Comment:
Needs to be A_TARGS := $(patsubst $(A_SRC_DIR)/%.jpg,$(A_DEST_DIR)/training/%.jpg,$(A_SRC))